src/App.jsx

```diff
@@ -2,14 +2,18 @@ import React from 'react'
 import Home from './Home'
 import Header from './components/Header'
 import { Switch, Route } from 'react-router-dom'
+import Footer from './components/Footer'
+import About from './components/About'
 
 const App = () => {
   return (
     <>
       <Header />
       <Switch>
         <Route exact path="/" component={Home}></Route>
+        <Route exact path="/about" component={About}></Route>
```

Comment:
Why have you added this?

Reply:
We thought that an about page was connected to the styling of the website but we saw your comment about out of scope so now its removed!